md/wildfly-bundle.md

```diff
@@ -40,61 +40,68 @@ The Wildfly bundle is based on a standard Wildfly installation with the followin
 * `standalone/deployments/bonita-all-in-one-[version].ear`: Bonita BPM Portal (web application) and EJB3 API.
 
 ::: info
-**Note:** Starting from Bonita BPM 7.3.0, Bonita BPM Platform configuration, including the license file, is stored in the same database as the Bonita BPM Engine data, namely in the `CONFIGURATION` table.  
+**Note:** Starting from Bonita BPM 7.3.0, Bonita BPM Platform configuration, including the license file, is stored in the same database as the Bonita BPM Engine data, namely in the `CONFIGURATION` table.
 To initialize and update this configuration, a [*Platform setup tool*](BonitaBPM_platform_setup.md) is provided and embedded in Bonita BPM bundles. 
-It will be launched automatically when you start the Wildfly bundle to initialize the database.  
+It will be launched automatically when you start the Wildfly bundle to initialize the database.
 :::
 
-So your bundle also contains:  
+So your bundle also contains:
 `setup`: database management for Bonita BPM Platform configuration and Bonita BPM Engine data, and a tool to update the configuration.
 
-<a id="configuration" />
 
-### Configure the Wildfly bundle
+### Get and install a license
 
 ::: info
-If you just want to try Bonita BPM Platform with the embedded h2 database (only for development phase of your project), you can skip the next two paragraphs.  
-For production, you are recommended to use one of the supported databases, with the following steps.
+This is not necessary if you are installing the Community edition.
```

Comment:
Change to say that it is mandatory for subscription edition instead or say in the section title plz